internal/cmd/group_utils.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/tursodatabase/turso-cli/internal"
+	"github.com/tursodatabase/turso-cli/internal/turso"
+)
+
+func ensureGroupAwake(client *turso.Client, groupName string) (bool, error) {
+	group, err := getGroup(client, groupName)
+	if err != nil {
+		return false, fmt.Errorf("failed to get group info: %w", err)
+	}
+
+	groupStatus := aggregateGroupStatus(group)
+	if groupStatus != "Archived" {
+		return true, nil
+	}
+
+	fmt.Printf("The group %s is currently archived. Do you want to wake it up? [Y/n]: ", internal.Emph(groupName))
```

Comment:
We already have a function to handle this logic, named `promptConfirmation`.
While you're at it if you can move it to the `prompt` package I would really appreciate it 😄